tests/gitlab/build_and_test

```diff
@@ -144,6 +144,11 @@ then
     echo "Removing existing configuration"
     make distclean
 
+    # Add -O2 flag for debug builds to speed up the tests.
+    sed -e 's/^\(DEBUG_FLAGS =.*\)$/\1 -O2/g' config/defaults.mk > \
```

Comment:
There could be `sed -i` for an inline change without the need to move files, but I guess you want to extra safe here. Hopefully, the scenario when sed does not produce any output and does not report an error (so the config would be overwritten by an empty file) is very unlikely.

Reply:
The `-i` option of `sed` is not exactly portable -- it behaves differently on macOS.

Reply:
Ahh, apple problems :apple: :smile: